WPF/View/ViewClasses/ToolTipData.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SmartPert.View.ViewClasses
+{
+    /// <summary>
+    /// Utility class for storing task data to be outputted in a string. 
+    /// Created 2/25/2021 by Tyler Kness-Miller
+    /// </summary>
+    class ToolTipData
+    {
+        //Private fields.
+        private string name;
+        private DateTime start;
+        private DateTime? end;
+        private int duration;
+        private int maxDuration;
+        private int minDuration;
+        private string description;
+
```

Comment:
I'm not convinced on data duplication, I think the ideal solution would be data binding to the tasks data.